OtripleS.Web.Api.Tests.Unit/Services/Foundations/Attachments/AttachmentServiceTests.Exceptions.RetrieveAll.cs

```diff
@@ -26,12 +26,12 @@ public void ShouldThrowDependencyExceptionOnRetrieveAllAttachmentsWhenSqlExcepti
                     .Throws(sqlException);
 
             // when
-            Action retrieveAllAttachmentsAction = () =>
-                this.attachmentService.RetrieveAllAttachments();
+             Action retrieveAllAttachmentAction = () =>
```

Comment:
Why space?

Reply:
sorry but which space ?

Reply:
All cleared now.

Reply:
still it saying changes requested.

Reply:
can u suggest me for this??

Reply:
Build is failing. Merge the latest master.

Reply:
Not working